packages/jest-jasmine2/package.json

```diff
@@ -19,7 +19,8 @@
     "jest-snapshot": "^22.4.0",
     "jest-util": "^22.4.1",
     "pretty-format": "^22.4.0",
-    "source-map-support": "^0.5.0"
+    "source-map-support": "^0.5.0",
+    "sprintf-js": "^1.1.1"
```

Comment:
do we need this when we have https://nodejs.org/api/util.html#util_util_format_format_args?

Reply:
Nice I didn't know about `util.format` just updated :)